core/controllers/suggestion_test.py

```diff
@@ -1389,6 +1263,157 @@ def test_cannot_update_question_when_provided_state_data_is_invalid(self):
             }, csrf_token=csrf_token, expected_status_int=400)
         self.logout()
 
+    def test_suggestion_creation_when_images_are_not_provided(self):
```

Comment:
Is it possible to perhaps add, to the end of these tests, something that retrieves the question and checks that e.g. the images resolve correctly? That way we can be certain that the suggestion has gone through correctly and come out properly at the other end.